front/lib/connectors_api.ts

```diff
@@ -31,6 +32,13 @@ export const ConnectorsAPI = {
 
     return _resultFromResponse(res);
   },
+
+  async getSyncStatus(connectorId: string): Promise<
+    ConnectorsAPIResponse<{
+      lastSyncStatus: ConnectorSyncStatus;
```

Comment:
If we have a lastSyncStatus should we hav ea lastSyncAttemptTime ? or lastSyncTime?

Reply:
I think sending the the `lastSyncStartTime` from the connectors database is the most reliable way of getting the last time we at least "tried to sync".
So I will add that.